src/components/Menu/MenuContent/MenuContent.tsx

```diff
@@ -17,9 +17,9 @@ const MenuContent: FC<MenuContentProps> = ({
 	const bem = bemCls.bind(root);
 	const rootCls = clsx(className, bem('item'), getVariantClasses(root, variants));
 
-	const renderIcon = (iconNode: ReactNode | string) => <span>{iconNode}</span>;
+	const renderIcon = (iconNode: ReactNode) => iconNode;
```

Comment:
Gaat die functie nog andere argumenten bevatten in de toekomst? If not, verwijderen en rechtstreeks `iconNode` renderen.

Reply:
Dit is gewoon lazy 🙈